pygasprice_client/__init__.py

```diff
@@ -78,7 +81,7 @@ def _fetch_price(self):
             if self._expired:
                 self.logger.info(f"Current gas prices from {self.URL} became available")
                 self._expired = False
-        except:
+        except (ConnectionError, HTTPError, RequestException) as ex:
```

Comment:
I think here we still want to catch all exceptions, otherwise the background thread will exit and will require a restart of process

Reply:
I reverted the change.